src/javascript/class-scanner.ts

```diff
@@ -211,6 +213,7 @@ export class ClassScanner implements JavaScriptScanner {
       astNode,
       properties,
       methods,
+      staticMethods,
```

Comment:
Yeah, that'd be a nice refactoring to do. This fell victim to some of the time pressure as we rushed to get things together.